rotkehlchen/tests/exchanges/test_coinbase.py

```diff
@@ -860,6 +860,218 @@ def test_asset_conversion_choosing_fee_asset():
+        },
+        {
+            "id": "REDACTED",
+            "type": "advanced_trade_fill",
+            "status": "completed",
+            "amount": {
+                "amount": "-1.120000",
+                "currency": "ETH"
+            },
+            "native_amount": {
+                "amount": "-3400.98",
+                "currency": "EUR"
+            },
+            "description": null,
+            "created_at": "2024-03-07T09:12:51Z",
+            "updated_at": "2024-03-07T09:12:51Z",
+            "resource": "transaction",
+            "resource_path": "/v2/accounts/REDACTED/transactions/REDACTED",
+            "instant_exchange": false,
+            "advanced_trade_fill": {
+                "fill_price": "3334.341",
+                "product_id": "ETH-USDC",
+                "order_id": "REDACTED",
+                "commission": "0.0000005",
+                "order_side": "sell"
+            },
+            "details": {
+                "title": "Filled ETH on sell",
+                "subtitle": null,
+                "header": null,
+                "health": "positive"
+            },
+            "hide_native_amount": false
+        },
+        {
+            "id": "REDACTED",
+            "type": "advanced_trade_fill",
+            "status": "completed",
+            "amount": {
+                "amount": "502.589290",
+                "currency": "USDC"
+            },
+            "native_amount": {
+                "amount": "461.08",
+                "currency": "EUR"
+            },
+            "description": null,
+            "created_at": "2024-03-07T09:12:51Z",
+            "updated_at": "2024-03-07T09:12:51Z",
+            "resource": "transaction",
+            "resource_path": "/v2/accounts/REDACTED/transactions/REDACTED",
+            "instant_exchange": false,
+            "advanced_trade_fill": {
+                "fill_price": "3400.12",
+                "product_id": "ETH-USDC",
+                "order_id": "REDACTED",
+                "commission": "0",
+                "order_side": "buy"
+            },
+            "details": {
+                "title": "Filled USDC on buy",
+                "subtitle": null,
+                "header": null,
+                "health": "positive"
+            },
+            "hide_native_amount": false
+        }]
+}
+"""
+
+    def mock_coinbase_query(url, **kwargs):  # pylint: disable=unused-argument
+        if 'transaction' in url:
+            if 'next-page' in url:
+                return MockResponse(200, TRANSACTIONS_RESPONSE)
+            # else
+            return MockResponse(200, mock_transactions_response)
+        if 'accounts' in url:
+            # keep it simple just return a single ID and ignore the rest of the fields
+            return MockResponse(200, '{"data": [{"id": "5fs23", "updated_at": "2020-06-08T02:32:16Z"}]}')  # noqa: E501
+        # else
+        raise AssertionError(f'Unexpected url {url} for test')
```

Comment:
can you define this as a `_function` in the file and use it in both places? Probably with a better name and a docstring explaining what it does